util/wren.mk

```diff
@@ -116,6 +116,8 @@ else
 	# Link in the right libraries needed by libuv on Windows and Linux.
 	ifeq ($(OS),mingw32)
 		LIBUV_LIBS := -lws2_32 -liphlpapi -lpsapi -luserenv
+	else ifeq ($(OS),freebsd12.0)
```

Comment:
is the `12.0` here very specific? wouldn't it limit other versions of freebsd (I'm unfamiliar with the os semantics, sorry!)?